### PR TITLE
Update ASCAT module

### PR DESCRIPTION
**Pull request by SusiJo**
## PR checklist

This PR implements the new `LogRCorrection` function of ASCAT 3.0 and fixes some issues. 
Additionally, the module was prepared to work with igenomes data.

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by SusiJo** [comment]
I'm still trying to prepare small testdata to properly test this module on GHA. So far, it's only possible to test it locally.

**Comment by SusiJo** [comment]
I've also added plots before and after `LogRCorrection`, `alleleFrequencies.txt` per chromosome and `metrics.txt` as output. If that is not desired, I can remove it again.

File: modules/ascat/main.nf [hunk 1]
```diff
@@ -11,13 +11,19 @@ process ASCAT {
     tuple val(meta), path(input_normal), path(index_normal), path(input_tumor), path(index_tumor)
     path(allele_files)
     path(loci_files)
+    path(gc_file)   // optional
+    path(rt_file)   // optional
+    path(bed_file)  // optional
+    path(ref_fasta) // optional
```
[the remark this answers is not in the document]
Comment (SusiJo) [reply modules/ascat/main.nf:17]: Do you mean changing `ref_fasta` to `fasta`? Or removing all opitonal inputs, except the fasta?

[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply modules/ascat/main.nf]: just `fast` would be more in line with the naming in the other modules

**Review by FriederikeHanssen — APPROVED** [review]
Great work, @SusiJo ! 🚀

File: modules/ascat/meta.yml [hunk 2]
```diff
@@ -1,13 +1,14 @@
 name: ascat
 description: copy number profiles of tumour cells.
 keywords:
-  - sort
+  - bam
```
[the remark this answers is not in the document]
Comment (SusiJo) [reply modules/ascat/meta.yml:4]: ah yes, good spot! I'll add that

**Pull request merged**